sdk/src/config/grpc_configuration.rs

```diff
@@ -6,6 +6,8 @@ pub struct GrpcConfiguration {
     /// The duration the client is willing to wait for an RPC to complete before it is terminated
     /// with a DeadlineExceeded error.
     pub(crate) deadline: Duration,
+    /// The number of grpc channels (TCP connections) to create
+    pub(crate) num_channels: u32,
```

Comment:
Nit: every other configuration in here uses full words - prefer “number.”
Nit: it is a number, it doesn’t need Hungarian notation to help clarify its type. Consider channel_count instead?

Reply:
this is an internal variable so if you prefer me to change it i will. in a vacuum I don't necessarily disagree with your naming preference, but in practice, my choice of names here were intended to align with the names of these fields in all of the other existing SDKs that already have support for this, and I probably have a slight pref for consistency with them vs. other naming considerations that only take into account this particular SDK. thoughts?

Reply:
if this is what was done in other sdks then consistency would be the right choice